File: backend/src/main/java/org/example/backend/domain/aquarium/dto/AquariumListResponseDto.java
package org.example.backend.domain.aquarium.dto

import org.example.backend.domain.aquarium.entity.Aquarium
import java.time.LocalDateTime

@JvmRecord
data class AquariumListResponseDto(
    val aquariumId: Long,
    val aquariumName: String,
    val createDate: LocalDateTime
) {
    constructor(aquarium: Aquarium) : this(
        aquarium.id,
        aquarium.name,
        aquarium.createDate
    )
}

File: backend/src/main/java/org/example/backend/domain/aquarium/dto/AquariumListResponseDto.kt
package org.example.backend.domain.aquarium.dto

import org.example.backend.domain.aquarium.entity.Aquarium
import java.time.LocalDateTime

@JvmRecord
data class AquariumListResponseDto(
    val aquariumId: Long?,
    val aquariumName: String?,
    val createDate: LocalDateTime?
) {
    constructor(aquarium: Aquarium) : this(
        aquarium.getId(),
        aquarium.name,
        aquarium.getCreateDate()
    )
}
